src/Imdb/Title.php

```diff
@@ -581,13 +581,14 @@ public function movie_recommendations()
                     $movie['title'] = utf8_decode(trim($get_link_and_name->item(0)->nodeValue));
                     $movie['imdbid'] = $ref[1];
                     $get_rating = $xp->query(".//span[contains(@class, 'ipc-rating-star--imdb')]", $cell);
-                    if (!empty($get_rating)) {
+                    if (!empty($get_rating) && $get_rating->item(0) != null) {
                         $movie['rating'] = trim($get_rating->item(0)->nodeValue);
                     } else {
                         $movie['rating'] = -1;
                     }
                     $getImage = $xp->query(".//div[contains(@class, 'ipc-media ipc-media--poster')]//img", $cell);
-                    $movie['img'] = $getImage->item(0)->getAttribute('src');
+                    if(!empty($getImage) && $getImage->item(0) != null && !empty($getImage->item(0)->getAttribute('src')))
+                        $movie['img'] = $getImage->item(0)->getAttribute('src');
```

Comment:
I'm not sure I see the value in this fixed size small image, but it would be better if 'img' were always set in the result as each of the other fields are. null seems an appropriate fallback value but it's been a long time since I wrote php so empty string would do it too I suppose.

Reply:
@tboothman 
Agree, if empty getAttribute src `$movie['img'] = ''`
I use that image in my application, so it has value for me although others may not be using this.